src/AboutPage/About.tsx

```diff
@@ -1,8 +1,28 @@
-import React from 'react'
-
 function About() {
   return (
-    <div>About</div>
+    <>
```

Comment:
Still need to change the text styling (colors, fonts, etc.). `main` contains the colors and fonts, so merge from main and do that.